controller/src/main/java/io/pravega/controller/server/ControllerService.java

```diff
@@ -383,6 +383,7 @@ public static long nextRequestId() {
                                   return CreateStreamStatus.newBuilder().setStatus(status).build();
                               }, executor);
                           } else {
+                              log.info("Stream {} already exists ", NameUtils.getScopedStreamName(scope, stream));
```

Comment:
Please add requestId to to the log message.

Reply:
Added request id in info log message.